src/services/user.methods.ts

```diff
@@ -0,0 +1,100 @@
+import { Router } from 'express';
+import { Op } from 'sequelize';
+import { userRouter } from '../controllers/user.routes';
+import { IBaseUser, IUser, UserModelAttributes } from "../interfaces/user.types";
+import { UserErrors } from '../models/user.model';
+import { initializeUserTable } from '../services/user.initialize';
+
+const users: IUser[] = [];
+
+const throwUserError = (type: UserErrors, param?: unknown): void => {
+    switch (type) {
+        case UserErrors.UserIdDoesNotExist:
+            throw new Error(`user with id ${param} doesn't exist`);
+        case UserErrors.UserRemoved:
+            throw new Error(`user with id ${param} exists but removed`);
+        case UserErrors.UserLoginDoesNotExist:
+            throw new Error(`user with login ${param} doesn't exist`)
+    }
+}
+
+export const createUser = async (userData: IBaseUser, users: UserModelAttributes): Promise<IUser | void> => {
+    const isUserExist = !!users.findOne({
+        where: {
+            login: userData.login
+        }
+    })
+
+    if (isUserExist) throw new Error(`user witn login ${userData.login} already exists`);
+
+    const newUser = users.build({ ...userData, isDeleted: false });
+
+    await newUser.save();
+
+    return newUser.toJSON();
+}
+
+export const editUser = async (userData: IUser, users: UserModelAttributes): Promise<IUser | void> => {
```

Comment:
Please use variables. It'll make your code more cleaner and readable.
For example:
```
const id = userData.id
...
        where: { id }
...
```